### PR TITLE
Fixed connections.js and smi_results.js components

### LINKED ISSUE #8675: [UI] Convert `smi_results.js` class-based components to functional component
### Current Behavior
`smi_results.js` is a class-based components.

### Desired Behavior
`smi_results.js` should be a functional component.

### Implementation
Here is a quick blog you can read : [Link](https://www.digitalocean.com/community/tutorials/five-ways-to-convert-react-class-components-to-functional-components-with-react-hooks)

### Mockups

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Layak-Ali**
**Notes for Reviewers**
Converted **connections.js** and **smi_results.js** class-based components to functional component

This PR fixes #8675 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Rajdip019** [comment]
The es-lint check has to pass. @Layak-Ali

File: ui/pages/smi_results.js [hunk 1]
```diff
@@ -11,27 +11,25 @@ const styles = { paper : { maxWidth : '90%',
   margin : 'auto',
   overflow : 'hidden', } };
 
-class SMIResults extends React.Component {
-  componentDidMount () {
+const SMIResults = ({ updatepagepath, classes }) => {
+  useEffect(() => {
     console.log(`path: ${getPath()}`);
```
Comment (Ckbhatia) [inline ui/pages/smi_results.js:16]: @Rajdip019 What's the reason to console log values project-wide?

File: ui/pages/smi_results.js [hunk 2]
```diff
@@ -11,27 +11,25 @@ const styles = { paper : { maxWidth : '90%',
...
+      </Head>
+      <Paper className={this.props.classes.paper}>
+        <MesherySMIResults />
+      </Paper>
+    </React.Fragment>
+  );
+};
 
-const mapDispatchToProps = dispatch => ({ updatepagepath : bindActionCreators(updatepagepath, dispatch) })
+const mapDispatchToProps = dispatch => ({ updatepagepath : bindActionCreators(updatepagepath, dispatch) });
```
Comment (Ckbhatia) [inline ui/pages/smi_results.js:32]: Please check your prettier configuration

File: ui/pages/smi_results.js (hunk 1 above)
Comment (Rajdip019) [reply ui/pages/smi_results.js:16]: We can just simply remove this.

**Comment by Rajdip019** [comment]
@Layak-Ali are you still working on it?

File: ui/pages/smi_results.js [hunk 3]
```diff
@@ -11,27 +11,25 @@ const styles = { paper : { maxWidth : '90%',
...
-      </React.Fragment>
-    );
-  }
-}
+  return (
+    <React.Fragment>
+      <Head>
+        <title>SMI Results | Meshery</title>
+      </Head>
+      <Paper className={this.props.classes.paper}>
```
Comment (sudhanshutech) [inline ui/pages/smi_results.js:25]: why using `this.`

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
@Layak-Ali please fix these small changes

**Comment by sudhanshutech** [comment]
// @ShravaniAK

**Pull request closed without merging**